crates/resolver/src/system_conf/unix.rs

```diff
@@ -85,7 +85,10 @@ fn into_resolver_config(
         });
     }
     if nameservers.is_empty() {
-        tracing::warn!("no nameservers found in config");
+        Err(io::Error::new(
+            io::ErrorKind::Other,
+            "No nameservers found in config",
```

Comment:
Nit: I think prevalent style is to all-lowercase error messages like this.

Reply:
Thanks for the quick review! So, I was going to leave it lowercase but I made it uppercase because the [error message below](https://github.com/hickory-dns/hickory-dns/pull/2327/files#diff-bd207ac182060e83598dd8cd393b88bfbcb4cca78c11b42583ddec40d46c0cc9R105) was capitalized. Let me know if I should just change this one or both to be more consistent with the other errors.

```rust
search.push(Name::from_str_relaxed(search_domain).map_err(|e| {
            io::Error::new(
                io::ErrorKind::Other,
                format!("Error parsing resolv.conf: {e}"),
            )
        })?);
```

Reply:
For now I just changed this one but am happy to update the other one too.